internal/tests/execute.go

```diff
@@ -0,0 +1,75 @@
+/*
+ * Flow CLI
+ *
+ * Copyright 2022 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package tests
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/onflow/flow-cli/internal/command"
+	"github.com/onflow/flow-cli/pkg/flowkit"
+	"github.com/onflow/flow-cli/pkg/flowkit/services"
+)
+
+type flagsTests struct {
+	// Nothing for now
+}
+
+var testFlags = flagsTests{}
+
+var ExecuteCommand = &command.Command{
+	Cmd: &cobra.Command{
+		Use:     "run <filename>",
+		Short:   "Run a test script",
+		Example: `flow tests run script.cdc`,
```

Comment:
I feel `flow test run` would be better name (following others). I also wonder if we could just have it `flow test` without the `run`? Do you think there is another test subcommand that will come in handy in the future? I feel that we could add features like coverage report, benchmarking etc all as flags.

Reply:
Actually, I also wanted to go with just `flow test`, but wasn't sure / didn't know if it's possible to drop the second portion of the command. If we can, that would be great.

Reply:
Added with f8dbe2e41249002d54d7fcf68ba1397d1ba8ba2b